docs/apis/cluster_execution.md

```diff
@@ -80,67 +80,73 @@ Note that the program contains custom user code and hence requires a JAR file wi
 the classes of the code attached. The constructor of the remote environment
 takes the path(s) to the JAR file(s).
 
-## Remote Executor
+## Linking with modules not contained in the binary distribution
 
-Similar to the RemoteEnvironment, the RemoteExecutor lets you execute
-Flink programs on a cluster directly. The remote executor accepts a
-*Plan* object, which describes the program as a single executable unit.
+The binary distribution contains jar packages in the `lib` folder that are automatically
+provided to the classpath of your distrbuted programs. Almost all of Flink classes are
+located there with a few exceptions, for example the streaming connectors and some freshly
+added modules. To run code depending on these modules you need to make them accessible
```

Comment:
Why listing the streaming connectors and not the other libraries? Maybe we can list/link all of them here.